### DISCUSSION TITLE
Consider adding a way to see raw anon scores

### DISCUSSION THREAD

**Discussion by zwrawr** [ludumdare/ludumdare#2059] [Ideas]
## Twitter thread
https://twitter.com/jharler_dev/status/1130966000839680000?s=19
> @jharler_dev
> #ldjam results came in.  Mine don't seem to line up with feedback.  I noticed a lot more ratings than comments (2x).  @mikekasprzak would it be possible to see an anon list of ratings you got?  I'm interesting in seeing if there were any all 1-star ratings mean to lower scores.

> @mikekasprzak 
> This is unfortunately the norm. Many first-time users don't realize there is actual benefit to leaving feedback. Also scores aren't a simple average.

> @jharler_dev
> Consider this a formal feature request for an anonymous list of ratings your entry received. I would be very interested in seeing how individual ratings look.

### Similar suggestion
see #991 
One option would be a way to see how many of each star level you got. Similar to what amazon does. 

I. E. 
4 :star:  :star:  :star:  :star:  :star:  
8 :star:  :star:  :star:  :star:  
1 :star:  :star:  :star:  
2 :star:  :star:   
1 :star:

**Comment by jharler** [comment]
It would be great to see the overview of votes as suggested, but it would also be great to see a list of all the votes you received to see how people voted as a whole (anonymous of course).  This would specifically allow you to isolate ratings that were possibly intended to lower your score (all 1-star votes).

**Comment by zwrawr** [comment]
see #991 for the amazon like view
